src/Test.UnitTests.Sarif.Multitool/GenericCommandTests.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using Xunit;
+
+namespace Sarif.Multitool.UnitTests
+{
+    public class GenericCommandTests
+    {
+        [Fact]
+        public void CommandsDoNotHaveConflictingShortNames()
```

Comment:
>CommandsDoNotHaveConflictingShortNames() [](start = 20, length = 40)

thanks for adding this test!

Reply:
No problem. =)

I fixed the one case and then thought there would likely be more and they would be easy to add in the future.